README.md

```diff
@@ -1,2 +1,119 @@
+
+---
+
+## **Getting Started**
+
+To run the LAI pipeline, set up your environment and install the required dependencies. Follow the step-by-step instructions below.
+
+---
+
+## **1. Pipeline Overview**
+
+This pipeline processes genomic data by:
+
+- **Phasing** haplotypes using **Eagle**
+- **Inferring local ancestry** using **RFMix v2**
+- **Extracting ancestry-specific allele frequencies** from phased and painted data using **Tractor**
+- **Generating a joint VCF with ancestry-specific calls** using **generate\_output\_vcf.py**
+
+---
+
+## **2. Installation & Setup**
+
+### **Step 1: Clone the Repository**
+
+```bash
+git clone https://github.com/broadinstitute/gnomad_local_ancestry.git
+cd gnomad_local_ancestry/batch
+```
+
+### **Step 2: Install Dependencies**
+
+The pipeline requires **Python 3**, **Hail**, and several additional tools. Install the necessary dependencies using:
+
+```bash
+pip install hail
+pip install numpy pandas
+```
+
+Make sure you have Eagle and RFMix v2 installed. You can find installation instructions and a toy dataset in the **[Tractor Tutorial](https://github.com/Atkinson-Lab/Tractor-tutorial/tree/main)**.
+
+---
+
+## **3. Running the LAI Pipeline**
+
+### **Step 1: Phasing**
+
+To phase your genotype data using **Eagle**, run:
+
+```bash
+eagle --vcf input_data.vcf.gz --out phased_data.vcf.gz
+```
+
+Refer to the **[Tractor wiki](https://github.com/Atkinson-Lab/Tractor-tutorial/tree/main)** for a detailed guide on phasing.
+
+### **Step 2: Local Ancestry Inference using RFMix**
+
+After phasing, run **RFMix v2** to infer local ancestry:
+
+```bash
+rfmix \
+  -f phased_data.vcf.gz \
+  -r reference_panel.vcf.gz \
+  -m samplemap.txt \
+  -g geneticmap.txt \
+  -o painted_lai \
+  --chromosome=22
+```
+
+See the **[Tractor wiki](https://github.com/Atkinson-Lab/Tractor-tutorial/tree/main)** for additional instructions on reference panels and sample maps.
+
+### **Step 3: Extracting Ancestry-Specific Allele Frequencies**
+
+Once local ancestry inference is complete, extract **ancestry-specific allele frequencies** using **Tractor**:
+
+```bash
+python3 extract_tracts.py \
+  --vcf phased_data.vcf.gz \
+  --msp painted_lai.msp.tsv \
+  --num-ancs 2
+```
+
+### **Step 4: Generating a Joint VCF with Ancestry-Specific Annotations**
+
+The `generate_lai_vcf` function calls **generate\_output\_vcf.py**, a standalone Python/Hail script. **Batch processing cannot be used** for this function, but `generate_lai_vcf` remains useful for extracting LAI-informed VCF outputs. This script outputs an annotated VCF containing ancestry-specific allele frequency data:
```

Comment:
We note batch processing cant be used but thats the case for the entire pipeline if you are not at the Broad. I think covering this up front is enough and we dont need to mention it again or we can write a separate paragraph discussing batch vs the pipeline w/o batch.